src/main/java/uk/gov/ch/service/update/impl/OverseasEntityManagingOfficersServiceImpl.java

```diff
@@ -27,7 +27,7 @@ public class OverseasEntityManagingOfficersServiceImpl implements OverseasEntity
     @Override
     public List<OverseasEntityManagingOfficerData> getOverseasEntityManagingOfficers(String companyNumber)
             throws ManagingOfficerCountNotFoundException {
-        DataMap dataMap = new DataMap.Builder(null).build();
+        DataMap dataMap = new DataMap.Builder().build();
```

Comment:
Would be nice if this line could please be changed to use the builder correctly, as it's being changed in this PR anyway to use the new, no argument constructor.

Reply:
Apologies, I thought I had this covered- fixed!